cmd/sti/main.go

```diff
@@ -59,14 +67,23 @@ func newCmdBuild(req *api.Request) *cobra.Command {
 		Short: "Build a new image",
 		Long:  "Build a new Docker image named <tag> from a source repository and base image.",
 		Run: func(cmd *cobra.Command, args []string) {
-			if len(args) < 3 {
+			// Attempt to restore the build command from the configuration file
+			config.Restore(req, cmd)
```

Comment:
doesn't this make it impossible to override stored args for the various flags like nocache, scriptsurl, etc?